app/models/user.rb

```diff
@@ -10,10 +10,10 @@ class User < ActiveRecord::Base
   # :confirmable, :lockable, :timeoutable and :omniauthable
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :trackable, :validatable
-  has_many :comments
-  has_many :favorites
-  has_many :wishlists
-  has_many :ratings
+  has_many :comments, dependent: :destroy
+  has_many :favorites, dependent: :destroy
+  has_many :wishlists, dependent: :destroy
```

Comment:
create has_many through relationship to access the favorite books and books in a wishlist directly